eng/SharedFramework.External.props

```diff
@@ -93,7 +93,7 @@
     These compilation references are necessary to compile netstandard2.0 assemblies which are in the shared framework.
     This references are part of Microsoft.NETCore.App, so are listed here as references to be used during compilation only.
   -->
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'netstandard2.1'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' or $(TargetFrameworks.Contains('netstandard2.0')) or '$(TargetFramework)' == 'netstandard2.1'">
```

Comment:
Changes in this file _should_ not be needed because the outer build doesn't actually care about references (just calls inner build to determine what's what). What goes South without these bits in the conditions?

Reply:
Reference resolution issue:
```
C:\gh\AspNetCore\eng\targets\ResolveReferences.targets(121,5): error : Cannot reference "Microsoft.Win32.Registry". This dependency is not in the shared framework. See docs/SharedFramework.md for instructions on how to modify what is in the shared framework. [C:\gh\AspNetCore\src\DataProtection\DataProtection\src\Microsoft.AspNetCore.DataProtection.csproj]
```

These properties are used to workaround the fact that these references are compile time only since they exist in the shared framework but not in the ref pack.